### PR TITLE
chore: add cuda image for pytorch

### PR DESCRIPTION
**Pull request by Reton2**

### CODE REVIEW

File: python38-torch-cuda/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM nvidia/cuda:11.6.0-base-ubuntu20.04
```
Comment (tomhobson) [inline python38-torch-cuda/Dockerfile:1]: 20.04 is not LTS and also not very recent :thinking:

**Review by tomhobson — CHANGES_REQUESTED** [review]

File: python38-torch-cuda/Dockerfile (hunk 1 above)
Comment (Reton2) [reply python38-torch-cuda/Dockerfile:1]: I'll have a look

Comment (Reton2) [reply python38-torch-cuda/Dockerfile:1]: Seems Cuda 11.6 isn't available with ubuntu 22.04
There's Cuda 11.7, let's see if that works

**Review by tomhobson — CHANGES_REQUESTED** [review]
Also use jx pipelines!!!!

File: python38-torch-cuda/Dockerfile (hunk 1 above)
Comment (Reton2) [reply python38-torch-cuda/Dockerfile:1]: Seems pytorch doesn't have a Cuda 11.7 version yet, either we'll have to build an image from base (a bit hard with cuda) or leave as is @tomhobson

Comment (tomhobson) [reply python38-torch-cuda/Dockerfile:1]: leave as is, python is a fckin nightmare for support isn't it

**Comment by Reton2** [comment]
Won't this go through jx pipelines? @tomhobson

**Comment by Reton2** [comment]
lol, where do i go for the jx-pipelines stuff?

File: python38-torch-cuda/Dockerfile (hunk 1 above)
Comment (Reton2) [reply python38-torch-cuda/Dockerfile:1]: python should be fine, but cuda is worse

**Comment by Reton2** [comment]
https://dashboard-jx.jx.mqube.build/ns-jx/spring-financial-group/Dockerfiles/HEAD/3
This does use jx-pipelines no? @tomhobson

**Comment by Reton2** [comment]
Oh, my bad
Hadn't seen the new stuff xD

**Review by tomhobson — CHANGES_REQUESTED** [review]
Make sure you add this to the release / pull request pipeline in the .lighthouse directory

**Comment by Reton2** [comment]
From what I remember, Archie set it up to iterate over each dir. Nowhere to add pipeline 
https://github.com/spring-financial-group/Dockerfiles/blob/8521c2135a6961ef07ed2a9d458f31acff2bcf89/.lighthouse/jenkins-x/release.yaml#L29

**Comment by tomhobson** [comment]
Cool
/lgtm

**Review by Skisocks — APPROVED** [review]

**Pull request merged**